medusa/providers/nzb/anizb.py

```diff
@@ -110,8 +98,8 @@ def search(self, search_strings, age=0, ep_obj=None):
 
                                 items.append(item)
                             except (AttributeError, TypeError, KeyError, ValueError, IndexError):
-                                logger.log('Failed parsing provider. Traceback: {0!r}'.format
-                                           (traceback.format_exc()), logger.ERROR)
+                                log.error('Failed parsing provider. Traceback: {0!r}',
```

Comment:
log.exception(). We can also decide to skip these now for the provicers. And change them in batch for all.